mlflow/pyfunc/model.py

```diff
@@ -705,6 +747,8 @@ class ModelFromDeploymentEndpoint(PythonModel):
     This class is particularly used for running evaluation against an MLflow Deployments endpoint.
     """
 
+    _skip_wrapping_predict = True
```

Comment:
Is any test fails when we remove this flag? It seems the type hint is pretty broad so curious what kind of inputs will be considered invalid🤔

Reply:
I didn't try, but the type hint is Union[...](normally inferred as Any) and pd.DataFrame should not be used together with other type hints, so we need this flag here. BTW type hints will be limited to list[...] in a [follow-up PR](https://github.com/mlflow/mlflow/pull/14171) so the type hint here is not valid.

Reply:
I see... not a blocker, but maybe we can use a decorator like `@disable_type_hint_validation` rather than instance attribute, as it makes the scope more clear (directly applied to the method rather than the entire class).

Reply:
This makes sense, while for ChatModel predict method is abstract method, so we cannot apply the decorator for users 🤔 Otherwise we'll need to override __init_subclass__ on ChatModel as well, which is a bit tricky. I'll keep this for now and see if we have better options